Upstream-Status: Backport [https://github.com/storaged-project/blivet-gui/commit/eb8ec968]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

From eb8ec968d1ee8b4b710568b1eb8536296f90751e Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Thu, 17 Feb 2022 11:16:22 -0800
Subject: [PATCH] Use symbolic list-add and edit- icons (Adwaita dropped old
 ones)

adwaita-icon-theme 42 no longer includes the non-symbolic icons
for these names. The choices are to switch to symbolic ones or
carry the non-symbolic ones downstream.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 blivetgui/list_actions.py      | 8 ++++----
 blivetgui/processing_window.py | 6 +++---
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/blivetgui/list_actions.py b/blivetgui/list_actions.py
index b7c1f9c..5388feb 100644
--- a/blivetgui/list_actions.py
+++ b/blivetgui/list_actions.py
@@ -48,10 +48,10 @@ def __init__(self, blivet_gui):
         self.blivet_gui = blivet_gui
 
         icon_theme = Gtk.IconTheme.get_default()  # pylint: disable=no-value-for-parameter
-        icon_add = Gtk.IconTheme.load_icon(icon_theme, "list-add", 16, 0)
-        icon_delete = Gtk.IconTheme.load_icon(icon_theme, "edit-delete", 16, 0)
-        icon_edit = Gtk.IconTheme.load_icon(icon_theme, "edit-select-all", 16, 0)
-        icon_misc = Gtk.IconTheme.load_icon(icon_theme, "edit-paste", 16, 0)
+        icon_add = Gtk.IconTheme.load_icon(icon_theme, "list-add-symbolic", 16, 0)
+        icon_delete = Gtk.IconTheme.load_icon(icon_theme, "edit-delete-symbolic", 16, 0)
+        icon_edit = Gtk.IconTheme.load_icon(icon_theme, "edit-select-all-symbolic", 16, 0)
+        icon_misc = Gtk.IconTheme.load_icon(icon_theme, "edit-paste-symbolic", 16, 0)
 
         self.action_icons = {"add": icon_add, "delete": icon_delete, "edit": icon_edit,
                              "misc": icon_misc}
diff --git a/blivetgui/processing_window.py b/blivetgui/processing_window.py
index c400f90..a020ae8 100644
--- a/blivetgui/processing_window.py
+++ b/blivetgui/processing_window.py
@@ -93,9 +93,9 @@ def add_action_view(self):
         """
 
         icon_theme = Gtk.IconTheme.get_default()  # pylint: disable=no-value-for-parameter
-        icon_add = Gtk.IconTheme.load_icon(icon_theme, "list-add", 16, 0)
-        icon_delete = Gtk.IconTheme.load_icon(icon_theme, "edit-delete", 16, 0)
-        icon_edit = Gtk.IconTheme.load_icon(icon_theme, "edit-select-all", 16, 0)
+        icon_add = Gtk.IconTheme.load_icon(icon_theme, "list-add-symbolic", 16, 0)
+        icon_delete = Gtk.IconTheme.load_icon(icon_theme, "edit-delete-symbolic", 16, 0)
+        icon_edit = Gtk.IconTheme.load_icon(icon_theme, "edit-select-all-symbolic", 16, 0)
 
         actions_list = Gtk.ListStore(GdkPixbuf.Pixbuf, str, GdkPixbuf.Pixbuf)
 
